### PR TITLE
berlin chainspec

### PR DESCRIPTION
**Pull request by tkstanczak**

### CODE REVIEW

File: src/Nethermind/Nethermind.Network/ForkInfo.cs [hunk 1]
```diff
@@ -51,6 +51,11 @@ public static ForkId CalculateForkId(ISpecProvider specProvider, long headNumber
             long next = 0;
             for (int i = 0; i < specProvider.TransitionBlocks.Length; i++)
             {
+                if (specProvider.TransitionBlocks[i] >= long.MaxValue - 100)
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Network/ForkInfo.cs:54]: Why this const? Can we name this const better?

File: src/Nethermind/Nethermind.Specs/RopstenSpecProvider.cs [hunk 2]
```diff
@@ -55,17 +55,23 @@ public IReleaseSpec GetSpec(long blockNumber)
             {
                 return Istanbul.Instance;
             }
+
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Specs/RopstenSpecProvider.cs:58]: Could we rewrite all those methods using TransitionBlocks ?

File: src/Nethermind/Nethermind.Specs/Forks/11_Berlin.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        public bool IsEip1052Enabled => true;
+        public bool IsEip1283Enabled => false;
+        public bool IsEip1234Enabled => true;
+        public bool IsEip1344Enabled => true;
+        public bool IsEip2028Enabled => true;
+        public bool IsEip152Enabled => true;
+        public bool IsEip1108Enabled => true;
+        public bool IsEip1884Enabled => true;
+        public bool IsEip2200Enabled => true;
+        public bool IsEip158IgnoredAccount(Address address) => false;
```
[the remark this answers is not in the document]
Comment (tkstanczak) [reply src/Nethermind/Nethermind.Specs/Forks/11_Berlin.cs:70]: nope => always false unless SystemAccount spec

File: src/Nethermind/Nethermind.Network/ForkInfo.cs (hunk 1 above)
Comment (tkstanczak) [reply src/Nethermind/Nethermind.Network/ForkInfo.cs:54]: will do

File: src/Nethermind/Nethermind.Specs/RopstenSpecProvider.cs (hunk 2 above)
Comment (tkstanczak) [reply src/Nethermind/Nethermind.Specs/RopstenSpecProvider.cs:58]: hmmm, transition blocks were added later, not 100% sure there would be no performance hit
out of the scope of this change

**Pull request closed without merging**